plugins/python-build/test/build/test.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+
+load ../test_helper
+export PATH="$BATS_TEST_DIRNAME/../../bin:$PATH"
+
+export PYTHON_BUILD_VERSION="${PYTHON_BUILD_VERSION:-3.8-dev}"
```

Comment:
"3.8-dev" 🤔 

This would mean pyenv CI will start failing if something bad has been pushed to CPython HEAD. Wouldn't it be sufficient to test some newer released version of CPython, e.g. 3.7.2....?

Reply:
Right now in the `.travis.yml` it is set to test 3.7.2 and 3.8-dev. That's a good point about spurious failures - we could configure [`allow_failures`](https://docs.travis-ci.com/user/customizing-the-build/#matching-jobs-with-allow_failures) to allow the 3.8-dev ones to fail without causing the whole build to appear to have failed, but still test it so we can see if there is actually something wrong.

Reply:
Gotcha. At least I am okay to run test against `3.8-dev`, and, with it we might be able to know potential issues with new release proactively.

Reply:
We now exclude failures that occur under 3.8-dev. Travis support helped also helped identify a way to exclude the duplicate Linux jobs. We will need to make sure to keep `osx_image` and `jobs.exclude` in-sync until it is fixed, but it's still better than having to explicitly mention all combinations, which will have more later.